fastlane/Fastfile

```diff
@@ -1159,6 +1164,45 @@ platform :ios do
     )
   end
 
+  private_lane :suppress_deprecation_warnings_in_headers do |options|
+    xcframework_path = options[:xcframework_path]
+    
+    # Find all RevenueCat-Swift.h files in the XCFramework
+    swift_headers = Dir.glob("#{xcframework_path}/**/RevenueCat-Swift.h")
+    
+    swift_headers.each do |header_path|
+      UI.message("Processing header: #{header_path}")
+      
+      # Read the current header content
+      content = File.read(header_path)
+      
+      # Check if pragma directives are already present
+      unless content.include?('#pragma clang diagnostic ignored "-Wdeprecated-declarations"')
+        # Add pragma directives to suppress StoreKit 1 deprecation warnings
+        # Insert after the existing pragma clang diagnostic push
```

Comment:
I think this comment is misleading. It's inserting after `-Wgcc-compat` right? Not a big deal though